.github/workflows/percy-issue-comment.yml

```diff
@@ -14,6 +14,18 @@ jobs:
       branch_name: ${{ fromJson(steps.fetch_pr.outputs.data).head.ref }}
       commit_sha: ${{ fromJson(steps.fetch_pr.outputs.data).head.sha }}
     steps:
+      - id: is_organization_member
```

Comment:
If we receive a contribution for the community will not work ? We will have to manually backport if needed ?